### PR TITLE
update description metadata - batch 9

### PR DESCRIPTION
**Pull request by v-kents**
@billwagner @tdykstra

Updates for missing/incorrect description metadata.

This is the last of the initial batch of scans - once these are all merged, I'll run another scan to make sure I didn't miss any.

### CODE REVIEW

File: docs/visual-basic/misc/bc33013.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: Operators cannot be declared '<keyword>"
```
Comment (tdykstra) [inline docs/visual-basic/misc/bc33013.md:2]: When the last word is surrounded by single-quotes, the closing single-quote is dropped in the description.

**Review by tdykstra — CHANGES_REQUESTED** [review]

File: docs/visual-basic/misc/bc33017.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: Conversion operators must be declared either 'Widening' or 'Narrowing"
```
Comment (Youssef1313) [inline docs/visual-basic/misc/bc33017.md:2]: There are still cases where there is a missing quote, like this one.

Comment (v-kents) [reply docs/visual-basic/misc/bc33017.md:2]: @Youssef1313 Yes, sorry. I was waiting for a possible fix on the scanning tool and haven't updated these yet. I hope to have news today.

File: docs/visual-basic/misc/bc33013.md (hunk 1 above)
Comment (v-kents) [reply docs/visual-basic/misc/bc33013.md]: fixed the ones I could find in this PR

File: docs/visual-basic/misc/bc33017.md (hunk 2 above)
Comment (v-kents) [reply docs/visual-basic/misc/bc33017.md]: fixed them manually after all

File: docs/visual-basic/misc/the-remote-server-machine-does-not-exist-or-is-unavailable.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: The remote server machine does not exist or is unavailable (Visual Basic)"
```
Comment (tdykstra) [inline docs/visual-basic/misc/the-remote-server-machine-does-not-exist-or-is-unavailable.md:2]: Why do some of these get (Visual Basic) appended to the description while others in the vb folder don't?

File: docs/visual-basic/programming-guide/language-features/arrays/how-to-sort-an-array.md [hunk 4]
```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: How to: sort an array in Visual Basic"
```
Comment (tdykstra) [inline docs/visual-basic/programming-guide/language-features/arrays/how-to-sort-an-array.md:2]: Why does this one get "in Visual Basic" while others get (Visual Basic)?

**Review by tdykstra — APPROVED** [review]
A couple of questions I'm curious about but nothing blocking.

File: docs/visual-basic/programming-guide/language-features/arrays/how-to-sort-an-array.md (hunk 4 above)
Comment (tdykstra) [reply docs/visual-basic/programming-guide/language-features/arrays/how-to-sort-an-array.md:2]: Never mind, I see now that the text is coming from the H1, not the metadata title.

File: docs/visual-basic/misc/the-remote-server-machine-does-not-exist-or-is-unavailable.md (hunk 3 above)
Comment (tdykstra) [reply docs/visual-basic/misc/the-remote-server-machine-does-not-exist-or-is-unavailable.md:2]: The text is coming from the H1, not the metadata title.

**Comment by v-kents** [comment]
@tdykstra Thank you for your patience with these. Now that they're all merged, I'll run one last pass to clean up anything missed. Coming soonish (if at all).

File: docs/visual-basic/misc/bc36016.md [hunk 5]
```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: Implicit variable '<variablename>' is invalid because of '<message>"
```
Comment (Youssef1313) [inline docs/visual-basic/misc/bc36016.md:2]: This is still missing the quote

File: docs/visual-basic/misc/bc41998.md [hunk 6]
```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: Statement recursively calls the containing 'AddHandler' for event '<eventname>"
```
Comment (Youssef1313) [inline docs/visual-basic/misc/bc41998.md:2]: This one as well.

File: docs/visual-basic/misc/method-methodname-has-no-parameter-named-parametername.md [hunk 7]
```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: Method '<methodname>' has no parameter named '<parametername>"
```
Comment (Youssef1313) [inline docs/visual-basic/misc/method-methodname-has-no-parameter-named-parametername.md:2]: And this

**Pull request merged**